src/applications/detection/main.cpp

```diff
@@ -297,6 +301,48 @@ int main(int argc, char** argv)
   std::ofstream outputFile;
   outputFile.open(outputFileName);
 
+#if USE_DEVIL
+  if( (ext == ".bmp") ||
+      (ext == ".gif") ||
+      (ext == ".jpg") ||
+      (ext == ".lbm") ||
+      (ext == ".pbm") ||
+      (ext == ".pgm") ||
+      (ext == ".png") ||
+      (ext == ".ppm") ||
+      (ext == ".tga") ||
+      (ext == ".tif") )
```

Comment:
Are these file formats not supported by opencv, which is already in the dependencies?